pkg/network/placement_configuration_test.go

```diff
@@ -49,95 +49,34 @@ var _ = Describe("Testing PlacementConfiguration", func() {
 				Infra:     defaultPlacementConfiguration.Infra,
 			},
 		}),
-		Entry("When the user hasn't explicitly changed PlacementConfiguration and a previous configuration exits should use the previous values on the current one", fillDefaultsCase{
-			previousConfig: &cnao.NetworkAddonsConfigSpec{
+		Entry("When Infra PlacementConfiguration is not nil should keep old values", fillDefaultsCase{
```

Comment:
The title doesn't describe what is happening imo, we have null PlacementConfiguration and update it with non-null values, and we keep the new values, right?

Reply:
yes you are correct. will fix
Done